src/components/Checkbox.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { LabelHTMLAttributes } from "react";
```

Comment:
LabelHTMLAttributes seems not used
```suggestion
import React from "react";
```

Reply:
Thank you!